### PR TITLE
Skip deadzone option on tilt

### LINKED ISSUE #10808: [Feature Request] Skipping analog stick deadzone on tilt input
I know about the existing deadzone option under Customize Tilt. But that option can only change the deadzone from an external controller (in this case, tilt input).
What I mean is a way to alter the initial jump (or skipping) the deadzone with a small tilt.

Perhaps I should explain with an example:
As you know, games react differently to how far the stick is pushed (i.e have different deadzones).
Now consider a racing game. As you know, in a racing game the car should react to even the slightest tilt. But because games have a deadzone of their own, you may see that the stick is pushed all the way near the edge but the car is still going straight! (like the following screenshot)
screenshot_2018-03-26-21-09-01-725_org ppsspp ppsspp

On the other hand, a game like NFS Carbon has a much smaller deadzone, but it's still enough to ruin the driving experience.
If there was a way we could skip this dead zone (in other words as soon as the phone is tilted a bit to the left or right, the control stick skips over the dead zone to that side) then the problem would be solved.

I think it only requires a constant number added to what is passed in from the input, so shouldn't be that hard.

### PR DESCRIPTION
**Pull request by iota97**
Closes #10808.

### CODE REVIEW

File: UI/TiltAnalogSettingsScreen.cpp [hunk 1]
```diff
@@ -41,6 +41,7 @@ void TiltAnalogSettingsScreen::CreateViews() {
 	settings->Add(new PopupSliderChoice(&g_Config.iTiltSensitivityX, 0, 100, co->T("Tilt Sensitivity along X axis"), screenManager(),"%"));
 	settings->Add(new PopupSliderChoice(&g_Config.iTiltSensitivityY, 0, 100, co->T("Tilt Sensitivity along Y axis"), screenManager(),"%"));
 	settings->Add(new PopupSliderChoiceFloat(&g_Config.fDeadzoneRadius, 0.0, 1.0, co->T("Deadzone Radius"), 0.01f, screenManager(),"/ 1.0"));
+	settings->Add(new PopupSliderChoiceFloat(&g_Config.fTiltDeadzoneSkip, 0.0, 1.0, co->T("Deadzone Skip"), 0.01f, screenManager(),"/ 1.0"));
```
Comment (unknownbrackets) [inline UI/TiltAnalogSettingsScreen.cpp]: Hm, I think the name is the trickiest part of this setting, as usual... I'd almost call this a "deadzone boost" or "title base radius".  It's hard to explain properly alongside "sensitivity"...

Skip makes it too easy to think it'll do the opposite of what it does.

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

File: UI/TiltAnalogSettingsScreen.cpp (hunk 1 above)
Comment (vnctdj) [reply UI/TiltAnalogSettingsScreen.cpp]: How about "alter deadzone on tilt"?

Comment (iota97) [reply UI/TiltAnalogSettingsScreen.cpp]: Changed with "Tilt Base Radius", using "deadzone" seem a source of confusion as if usually do the opposite indeed, need "lifezone" I guess :)

Comment (vnctdj) [reply UI/TiltAnalogSettingsScreen.cpp]: I'm having trouble understanding the meaning of "tilt base radius": what is the base here?

Comment (iota97) [reply UI/TiltAnalogSettingsScreen.cpp]: Base as a starting point, as when you move a little you start moving from the base radius (is a square, but still). TBH I'm not even sure if there is any non confusing way to say it.

**Pull request merged**